hs628_tools/random-forest.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+
+def rf(features, target, model='classifier'):
+    train_features, train_target, test_features, test_target = train_test_split(features, target, test_size=0.25, random_state=13)
+    if model == 'classifier':
+        clf = RandomForestClassifier(n_estimators=10, criterion='gini', max_features='auto',
```

Comment:
Arguments to the RandomForestClassifier are currently hardcoded, and while they may have been well chosen for the current implementation, it will be helpful to provide a way for having a way to get user input that can update these argument values, so that your implementation can be reusable.